ds18b20/ds18b20.go

```diff
@@ -33,6 +33,17 @@ func ConvertAll(o onewire.Bus, maxResolutionBits int) error {
 	return nil
 }
 
+// StartAll starts a conversion on all DS18B20 devices on the bus.
```

Comment:
FTR, I don't understand ConvertAll() either, since the resolution cannot be set, I don't understand the purpose of this function.

Reply:
Basically it's for savings of bandwidth. Settings are retained inside ds18b20. There is command to start conversion on all devices on the bus (can be up to 64) at once. What I usually do is: after issuing conversion start command I'm leaving the bus and returning later to collect data. When all data is collected on this particular bus I send conversion start command again.

Reply:
ConvertAll resolution setting is just to determine how long conversion is going to take and wait for this time. Resolution is programmed into ds18b20 during initialisation at function New.

Reply:
ok